config/settings/development.yml

```diff
@@ -1,14 +1,14 @@
 dfe_signin:
```

Comment:
This file is for local development, not the `qa` environment so the localhost URL's need to stay

Reply:
reverted changes made to development.yml file